caffe2/python/parallel_workers.py

```diff
@@ -110,7 +109,7 @@ def put_metric(self, key, value, count=True):
 
 
 class State():
-    six.add_metaclass(ABCMeta)
+    _metaclass__ = ABCMeta
```

Comment:
There is a typo in here, it should be `__metaclass__`

Reply:
Fixed.